tests/test_entity_identities.py

```
@@ -0,0 +1,757 @@
+            "merged_into": None,
+        }
+        graph.entities["entity:tools:python"] = {
+            "id": "entity:tools:python",
+            "slug": "python",
+            "category": "tools",
+            "aliases": [],
+            "merged_into": None,
+        }
+        auto_merge, _review = find_merge_candidates(graph)
+        assert len(auto_merge) == 0
+        assert len(review) == 0
```

Ruff (0.15.5)

[error] Undefined name `review`

(F821)